cfme/containers/provider/openshift.py

```diff
@@ -137,6 +158,23 @@ def __init__(
             endpoints=endpoints,
             appliance=appliance)
 
+    def create(self, **kwargs):
+
+        # Enable alerts collection before adding the provider to avoid missing active
+        # alert after adding the provider
+        # For more info: https://bugzilla.redhat.com/show_bug.cgi?id=1514950
+        if getattr(self, "alerts_type", None) == "Prometheus":
+            alert_profiles = self.appliance.collections.alert_profiles
```

Comment:
Thanks for being open and flexible about this, I prefer this much more to the layered methods that were in use before.  This makes it very clear what is being configured, and why.

Reply:
sure ;)